common/planners/src/plan_expression_monotonicity.rs

```diff
@@ -52,30 +54,27 @@ use crate::Recursion;
 pub struct ExpressionMonotonicityVisitor {
     input_schema: DataSchemaRef,
 
-    // the variable range left, we assume only one variable.
```

Comment:
> The comment of this struct, line 52 has "// Notice!! the mechanism doesn't solve multiple variables case." may need update since we now support multi-variable.

Yes, I will fix.